plugins/sdm/accessbot.py

```diff
@@ -122,7 +125,8 @@ def show_resources(self, message, match):
         """
         if not self._platform.can_show_resources(message):
             return
-        filter = self.extract_filter(message.body)
+        flags = FlagsHelper.extract_flags(message.body)
```

Comment:
Please leave a TODO for making this compatible with the approach followed in access_resource
```
# ...
flags = FlagsHelper.extract_flags(arguments, validators=ResourceGrantHelper.get_flags_validators())
yield from self.get_resource_grant_helper().request_access(message, resource_name, flags=flags)
```